GitCliUtil.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/config"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+type GitUtil struct {
+}
+
+func NewGitUtil() *GitUtil {
+	return &GitUtil{
+	}
+}
+
+const GIT_AKS_PASS = "/git-ask-pass.sh"
+
+func (impl *GitUtil) Fetch(rootDir string, username string, password string) (response, errMsg string, err error) {
+	log.Println(devtron, "git fetch ", "location", rootDir)
```

Comment:
lfs support add kar dete hain

Reply:
definitely a good feature to support. creating a separate issue for it. 
It would require significant testing, go-git support is unknown